CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This is all directly copied, except line 37 where info@gravitational.com is filled in.